budgie-advanced-brightness-controller/src/Applet.vala

```diff
@@ -0,0 +1,147 @@
+/*
+ * AdvancedBrightnessController 
+ * This file is part of UbuntuBudgie
+ * 
+ * Author: Serdar ŞEN github.com/serdarsen
+ * 
+ * Copyright © 2015-2017 Budgie Desktop Developers

```

Comment:
doubtful anything here is copyright for Budgie Desktop Developers.  Suggest delete

Reply:
Your welcome :) the headers changed.